### ISSUE TITLE
Time Picker Code Samples

### ISSUE THREAD

**Issue by manojdcoder** [NativeScript/nativescript-sdk-examples-ng#172]
- The HTML and TypeScript code in documentation (http://docs.nativescript.org/angular/code-samples/ui/time-picker) are pointing to something different than Time Picker.

<img width="1049" alt="timepicker" src="https://cloud.githubusercontent.com/assets/4987027/24346348/87f1bce6-12f1-11e7-9846-ae42501d077b.png">

- Please add examples for min and max time

**Comment by glorindel** [comment]
please can you revise this section ...

**Comment by sm1ddy** [comment]
I'd like to second this, the TimePicker samples are generally very hard to follow. I'm using vanilla javascript and can't even set one up using two-way binding to a viewModel. :\

**Comment by NickIliev** [comment]
Resolved via [this PR](https://github.com/NativeScript/nativescript-sdk-examples-ng/pull/234)
